tests/test_routes.py

```diff
@@ -0,0 +1,53 @@
+def test_get_one_planet(client, two_saved_planets):
+    # Act
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == {
+        "id": 1,
+        "name": "Mars",
+        "description": "Smallest planet"
+    }
+
+
+def test_get_one_planet_no_data_returns_404(client):
+    # Act
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 404
+
+
+def test_get_all_planets(client, two_saved_planets):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == [{
+        "id": 1,
+        "name": "Mars",
+        "description": "Smallest planet"
+        },
+        {
+        "id": 2,
+        "name": "Jupiter",
+        "description": "Largest planet"
+        }]
+    
+
+def test_create_one_planet(client):
+    # Act
+    response = client.post("/planets", json={
+        "name": "Earth",
+        "description": "Water planet"
+    })
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 201
+    assert response_body == "Planet Earth successfully created"
```

Comment:
In addition to using the response to write assertions, your test would be more robust if after the post request, you query the database for the object you just created. Since we run fixtures before each test, we know that before your test runs we will have a database with empty tables, after your post request in the test we will have one planet record. We can query for that one record by assuming it's id will be 1 and compare its attributes to the request body you sent with the post request. Like [this](https://github.com/Ada-C19/flasky/blob/zoisite/tests/test_routes.py#L31-L36)